packages/babel-plugin-ima-logger/rollup.config.js

```diff
@@ -0,0 +1,9 @@
+import common from '../../rollup.config.common';
+
+export default Object.assign({
+  external: ['@ima/plugin-logger'],
+  output: {
+   file: 'dist/main.js',
+   format: 'js'
+ }
+}, common);

```

Comment:
I would switch the arguments for Object.assign, since this way, you cannot override common settings, you can only extend it with new keys. `Object.assign({}, common, { external: []... })`

Reply:
It's possible to override common settings now.